app.rb

```diff
@@ -0,0 +1,59 @@
+require 'sinatra/activerecord'
+require 'sinatra/base'
+require 'sinatra/reloader'
+
+require_relative './lib/models/report'
+
+class App < Sinatra::Base
+  CHARTS = %i[authentications alb_errors].freeze
+
+  configure :development do
+    register Sinatra::Reloader
+    ActiveRecord::Base.logger&.level = :info
+  end
+
+  configure :test do
+    ActiveRecord::Base.logger&.level = :warn
+  end
+
+  configure :production do
+    ActiveRecord::Base.logger&.level = :info
+  end
+
+  get '/' do
+    chart_index = Time.now.to_i / 60 % CHARTS.length
```

Comment:
should we make sure this is UTC? `Time.now.utc`? Or set up `Time.zone = ActiveSupport::TimeZone['UTC']` if we want to use activesupport

Reply:
Yeah, originally I was trying to pull in ActiveSupport but looks like I'm gonna have to. Once I do that I'll go around and fix the time zones for all of these.

Reply:
I agree that most ActiveSupport magic is bloaty, but if we just did `require 'active_support/time'` I think we could get most of the benefits